Combinatorial_map/include/CGAL/internal/Combinatorial_map_utility.h

```diff
@@ -593,6 +594,18 @@ namespace CGAL
       };
   };
 
+  // Helper class to define container type depending on the Concurrent_tag
+  template<typename Concurrent_tag, class T, class Alloc_>
+  struct Container_type
+  {
+    typedef CGAL::Compact_container<T, Alloc_> type;
+  };
+  template<class T, class Alloc_>
+  struct Container_type<CGAL::Tag_true, T, Alloc_>
+  {
+    typedef CGAL::Concurrent_compact_container<T, Alloc_> type;
+  };
```

Comment:
In Triangulation_3 and Mesh_3, the tag `Parallel_tag` is used to decide if the data structures and algorithm must be parallel or not. Here you use `Tag_true` instead. Do you plan to document it that way?

Reply:
This class is internal and undocumented. I don't plan to change it for now.
Moreover, here the tag is only to choose between Compact_container or Concurrent_compact_container.

However you right, using a Concurrent_compact_container means probably parallel that it will be used in a parallel algorithm. Thus if you want I can change Tag_true by Parallel_tag.